source/configy/attributes.d

```diff
@@ -260,7 +260,7 @@ public struct SetInfo (T)
 
 public struct Converter (T)
 {
-    import dyaml.node;
```

Comment:
This is a breaking change as the type of `Node` changed from D-YAML to `config.backend.node`